packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -556,9 +570,15 @@ mixin WidgetsBinding on BindingBase, SchedulerBinding, GestureBinding, RendererB
   /// Whether the Flutter engine has rasterized the first frame.
   ///
   /// {@macro flutter.frame_rasterized_vs_presented}
-  Future<void> get firstFrameRasterized => _firstFrameCompleter.future;
+  bool get firstFrameRasterized => _firstFrameCompleter.isCompleted;
```

Comment:
Add a "see also" section linking to the future version?

Reply:
Done.